app/components/Dropdown/index.js

```diff
@@ -85,7 +85,7 @@ function List({ children }: { children: any }) {
   return <ul className={styles.dropdownList}>{children}</ul>;
 }
 
-function ListItem(props) {
+function ListItem(props: any) {
```

Comment:
helpful ;o

Reply:
Nah. The new version of flow failed without this... :/